benchmarks/src/main/scala/zio/redis/GetBenchmarks.scala

```diff
@@ -0,0 +1,59 @@
+package zio.redis
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+import zio.ZIO
+
+import scala.util.Random
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput, Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.SECONDS)
+@Measurement(iterations = 15, timeUnit = TimeUnit.SECONDS, time = 3)
+@Warmup(iterations = 15, timeUnit = TimeUnit.SECONDS, time = 3)
+@Fork(4)
+class GetBenchmarks {
+  import BenchmarkRuntime._
+  import BenchmarksUtils._
+
+  @Param(Array("500"))
+  private var count: Int = _
+
+  private var items: List[String] = _
+
+  @Setup(Level.Trial)
+  def setup(): Unit = {
+    items = (0 to count).toList.map(Random.nextString)
+    zioQuery(ZIO.foreach_(items)(i => set(i, i, None, None, None)))
```

Comment:
```suggestion
    zioQuery(ZIO.foreach_(items)(i => set(i, i)))
```

Those nones were required with the old API. This should be adjusted in the other benchmark as well.

Reply:
Done!